ddtrace/propagation/http.py

```diff
@@ -554,6 +564,9 @@ def _get_traceparent_values(tp):
         version, trace_id_hex, span_id_hex, trace_flags_hex = tp.split("-")
         # check version is a valid hexadecimal, if not it's invalid we will move on to the next prop method
         int(version, 16)
+        # https://www.w3.org/TR/trace-context/#version
+        if version == "ff":
+            raise ValueError("'ff' is an invalid traceparent version")
```

Comment:
Do we catch this exception in our library? If not, we should log a error and stop processing instead

Reply:
It's caught by the parent